<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>THandEyeCali3D</class>
 <widget class="QWidget" name="THandEyeCali3D">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>THandEyeCali3D</string>
  </property>
  <widget class="QVTKWidget" name="vtk_cad">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>371</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>step3:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>step6:</string>
   </property>
  </widget>
  <widget class="QVTKWidget" name="vtk_ret">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>290</y>
     <width>371</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_roi_cut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>ROI裁剪</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_get_pointcloud">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>获取点云</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>step4:</string>
   </property>
  </widget>
  <widget class="QVTKWidget" name="vtk_cam">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>371</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QVTKWidget" name="vtk_realtime">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>290</y>
     <width>371</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>step1:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>step2:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_registration">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>点云配准</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cal_hadeye_mat">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>计算手眼矩阵</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_load_CAD">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>加载CAD点云</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>step5:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save_cali_data">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保存标定数据</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_test">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>480</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>test</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
